src/equations/ideal_glm_mhd_2d.jl

```diff
@@ -196,19 +196,13 @@ end
     flux_nonconservative_powell(u_ll, u_rr, orientation::Integer,
                                 equations::IdealGlmMhdEquations2D)
     flux_nonconservative_powell(u_ll, u_rr,
-                                normal_direction_ll     ::AbstractVector,
-                                normal_direction_average::AbstractVector,
+                                normal_direction::AbstractVector,
                                 equations::IdealGlmMhdEquations2D)
 
 Non-symmetric two-point flux discretizing the nonconservative (source) term of
 Powell and the Galilean nonconservative term associated with the GLM multiplier
 of the [`IdealGlmMhdEquations2D`](@ref).
 
-On curvilinear meshes, this nonconservative flux depends on both the
-contravariant vector (normal direction) at the current node and the averaged
-one. This is different from numerical fluxes used to discretize conservative
-terms.
-
 ## References
```

Comment:
It should be mentioned here that this term is not exactly equivalent to the one proposed by Bohm et al., as the "metrics dealiasing" is done in a different manner.

Reply:
I added an additional comment about this to the docstring in https://github.com/trixi-framework/Trixi.jl/pull/2062/commits/95d846a989af7e2b03ffaf3fd49b82f9ea24b057